scripts/onyxia-set-repositories.sh

```diff
@@ -13,19 +21,13 @@ if [  "`which pip`" != "" ]; then
     fi
 fi
 
-if [  "`which conda`" != "" ]; then
-    if [[ -n "$CONDA_REPOSITORY" ]]; then
-        echo "configuration conda (add channels)"
-        conda config --add channels $CONDA_REPOSITORY
-        conda config --remove channels conda-forge
-        conda config --remove channels conda-forge --file /opt/conda/.condarc
+if command -v uv; then
+    if [[ -n "$PIP_REPOSITORY" ]]; then
+        echo "export UV_DEFAULT_INDEX=$PIP_REPOSITORY" >> "$ENV_FILE"
+        echo 'export UV_NATIVE_TLS=true' >> "$ENV_FILE"   
```

Comment:
Need a fi somewhere, I think. I got this while testing : 

> /opt/onyxia-set-repositories.sh: line 59: syntax error: unexpected end of file

Reply:
Good catch, thank's